MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationIndicatorLayerTest.java

```diff
@@ -114,6 +114,19 @@ public void testImageTiltDisplacementAsConstant() {
     assertEquals(layer.getImageTiltDisplacement().getValue(), propertyValue);
   }
 
+  @Test
+  @UiThreadTest
+  public void testImagePitchDisplacementAsConstant() {
+    Timber.i("image-tilt-displacement");
+    assertNotNull(layer);
+    assertNull(layer.getImagePitchDisplacement().getValue());
+
+    // Set and Get
+    Float propertyValue = 0.3f;
+    layer.setProperties(imageTiltDisplacement(propertyValue));
+    assertEquals(layer.getImagePitchDisplacement().getValue(), propertyValue);
```

Comment:
```suggestion
    assertEquals(propertyValue, layer.getImagePitchDisplacement().getValue());
```

Reply:
Expected value should be the first param.

Reply:
:+1: Totally agree, we should fix all test cases in this test for consistency.